buildkit/pg_partman.yaml

```diff
@@ -7,8 +7,9 @@ source: https://github.com/pgpartman/pg_partman/archive/refs/tags/v5.0.0.tar.gz
 description: Partition management extension for PostgreSQL.
 licence: PostgreSQL
 keywords:
-  - partition manager
-  - partition sets
+  - partition
+  - manager
```

Comment:
Merely having the manager or sets as keywords do not make sense

Reply:
I agree. Manager is not a useful tag.